packages/editor/src/components/theme-support-check/index.js

```diff
@@ -23,7 +18,7 @@ export function ThemeSupportCheck( {
 	const isSupported = (
 		Array.isArray( supportKeys ) ? supportKeys : [ supportKeys ]
 	 ).some( ( key ) => {
-		const supported = get( themeSupports, [ key ], false );
+		const supported = themeSupports[ key ] ?? false;
```

Comment:
`ThemeSupportCheck` is public API so we shouldn't change the behavior to require an object. We could either guard for this or add a default, no?

Reply:
Makes sense to me 👍 Added a guard in 5a792f778f00970d565ff615faf0f6b5560b2f42. Let me know what you think. Thanks 🙌

Reply:
Thank you!